### DISCUSSION TITLE
[google-apps-script] Slide class type missing setSkipped() function

### DISCUSSION THREAD

**Discussion by jkeegan2** [DefinitelyTyped/DefinitelyTyped#61119] [Issues with a @types package] [answered]
According to the documentation here:
https://developers.google.com/apps-script/reference/slides/slide#setSkipped(Boolean)

The slide class contains a method setSkipped(isSkipped: boolean). This is missing from the current type definitions.

[a comment by an automated account is not shown]

**Comment by mtgto** [comment, marked as answer]
@jkeegan2 It appears that you are correct.

Could you create a pull request to add setSkipped? We welcome it 😉

**Reply by jkeegan2** [reply]
Can do. Are there any tests I need to change? The tests folder is hard to understand but I don't see a lot of functionality being tested in there.

**Reply by mtgto** [reply]
> Can do.

It happy for us!

> Are there any tests I need to change? The tests folder is hard to understand but I don't see a lot of functionality being tested in there.

Yes, `test/google-apps-script-tests.ts` is messed... It seems to there is no strict rule 😇 

You can add a single line to EOL.
Ex: `SlidesApp.getActivePresentation().getSlides()[0].setSkipped(true);`
